setup.py

```diff
@@ -91,13 +91,14 @@
                         ]},
       scripts=glob('scripts/*'),
       extras_require={'test': ["nose >= 0.10.1", "pep8", 'mock'],
-                      'doc': ["Sphinx >= 1.2.2", "sphinx-bootstrap-theme"]},
+                      'doc': ["Sphinx >= 1.2.2"]},
```

Comment:
Great, once tests pass this is ready for merge!

On (Aug-24-15|15:57), Jose Navas wrote:
> > @@ -91,13 +91,14 @@
> >                          ]},
> >        scripts=glob('scripts/*'),
> >        extras_require={'test': ["nose >= 0.10.1", "pep8", 'mock'],
> > -                      'doc': ["Sphinx >= 1.2.2", "sphinx-bootstrap-theme"]},
> > +                      'doc': ["Sphinx >= 1.2.2"]},
> 
> Done
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1438/files#r37815230